### PR TITLE
Tweak webpack compile script

### PR DESCRIPTION
**Pull request by e1emeb0t**
当前的输出结果:

| Asset      | Size     | Optimized |
|---------- |-------|------- |
| a6393e88a363.svg  |   2 kB   | - |
| a61be9cda527.eot | 13.5 kB | - |
| b02bdc1b846f.ttf | 13.2 kB | - |
| 09162bc124a0.svg | 17.4 kB | - |
| 192bf63947be.svg | 2.13 kB | - |
| d2f69a92faa6.woff  | 8 kB | - |
| 711bbe69c0d5.svg | 1.73 kB | - |
| 7f42f1a38e98.svg | 2.08 kB | - |
| 50e4091cc60a.png | 30.3 kB | - |
| bundle.js  | 2.42 MB | 2.34 MB |
| bundle.css  |  108 kB | - |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "start": "cd site && node run.js",
     "test": "jest --config jest.json",
-    "prepublish": "npm run lint && npm run build",
-    "build": "cd build/site && webpack --progress",
+    "build": "rm -fr dist && cd build/site && webpack --progress",
```
Comment (rlog) [inline package.json]: 加上 `-p`  https://webpack.github.io/docs/cli.html#production-shortcut-p

Comment (e1emeb0t) [inline package.json]: 优化方式放到了`webpack.config.js`, 我把`--optimize-occurrence-order`补上.
https://github.com/webpack/docs/wiki/optimization#minimize

**Comment by e1emeb0t** [comment]
现在的`webpack`做了`code split`, 但是会卡住.

```shell
> rm -fr dist && cd build/site && webpack --progress

 78% additional chunk assets
```

**Comment by e1emeb0t** [comment]
看来需要`webpack@2.x`才可以解决ES6的code split问题, 暂时先不做code split了.

**Pull request closed without merging**